docker/istio-dev/README.ubi9.md

```diff
@@ -0,0 +1,205 @@
+# Istio Development Environment in Rootless Podman
```

Comment:
istio-dev is totally unmaintained and probably broken. I was thinking of submitting a PR to remove it entirely. Is there value? Are you using it?

Reply:
I am not using it. I realized that was out of date in the middle of my work on this PR.
I tried to start from a simple KIND setup. I can sync up istio repo kind-provisoiner later.

+1 I agree it's better removing istio-dev deprecated directory.